README.md

```diff
@@ -1 +1,186 @@
+
+-->
+# Nexus Repository apk Format
+
+[![Maven Central](https://img.shields.io/maven-central/v/org.sonatype.nexus.plugins/nexus-repository-apk.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:%22org.sonatype.nexus.plugins%22%20AND%20a:%22nexus-repository-apk%22) [![CircleCI](https://circleci.com/gh/sonatype-nexus-community/nexus-repository-apk.svg?style=svg)](https://circleci.com/gh/sonatype-nexus-community/nexus-repository-apk) [![Join the chat at https://gitter.im/sonatype/nexus-developers](https://badges.gitter.im/sonatype/nexus-developers.svg)](https://gitter.im/sonatype/nexus-developers?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [![DepShield Badge](https://depshield.sonatype.org/badges/sonatype-nexus-community/nexus-repository-apk/depshield.svg)](https://depshield.github.io)
+
+# Table Of Contents
+
+* [Developing](#developing)
+   * [Requirements](#requirements)
+   * [Building](#building)
+* [Using APK with Nexus Repository Manager 3](#using-apk-with-nexus-repository-manager-3)
+* [Compatibility with Nexus Repository Manager 3 Versions](#compatibility-with-nexus-repository-manager-3-versions)
+* [Installing the plugin](#installing-the-plugin)
+   * [Easiest Install](#easiest-install)
+   * [Temporary Install](#temporary-install)
+   * [(more) Permanent Install](#more-permanent-install)
+   * [(most) Permanent Install](#most-permanent-install)
+* [The Fine Print](#the-fine-print)
+* [Getting Help](#getting-help)
+
+## Developing
+
+### Requirements
+
+* [Apache Maven 3.3.3+](https://maven.apache.org/install.html)
+* [Java 8+](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+* Network access to https://repository.sonatype.org/content/groups/sonatype-public-grid
+
+Also, there is a good amount of information available at [Bundle Development](https://help.sonatype.com/display/NXRM3/Bundle+Development)
+
+### Building
+
+To build the project and generate the bundle use Maven
+
+    mvn clean package -PbuildKar
+
+If everything checks out, the bundle for apk should be available in the `target` folder
+
+#### Build with Docker
+
+`docker build -t nexus-repository-apk:0.0.1 .`
+
+#### Run as a Docker container
+
+`docker run -d -p 8081:8081 --name nexus nexus-repository-apk:0.0.1` 
+
+For further information like how to persist volumes check out [the GitHub repo for our official image](https://github.com/sonatype/docker-nexus3).
+
+The application will now be available from your browser at http://localhost:8081
+
+## Using APK With Nexus Repository Manager 3
+
+[We have detailed instructions on how to get started here!](docs/APK_USER_DOCUMENTATION.md)
+
+## Compatibility with Nexus Repository Manager 3 Versions
+
+The table below outlines what version of Nexus Repository the plugin was built against
+
+| Plugin Version | Nexus Repository Version |
+|----------------|--------------------------|
+| v0.0.1         | 3.19.0-SNAPSHOT          |
```

Comment:
NOTE: I'm surprised this isn't 3.19.0-1 or 3.20.0-SNAPSHOT given our recent release.  I believe you, but just surprised (thus NOTE).

Reply:
Great note! I was unsure myself, @DarthHater what should i be putting for version #? The POMs all reference 3.19.0-SNAPSHOT: 
![image](https://user-images.githubusercontent.com/10136383/66059656-393c1180-e4f1-11e9-8664-b79c25f81751.png)

Reply:
We need to sort that out. Now that 3.19.0-01 is out, I think the best thing to do would be to cut a release that uses that, and then work with @bhamail to figure out how we want SNAPSHOT stuff to be in the repos.